rundeckapp/grails-app/controllers/rundeck/controllers/ApiController.groovy

```diff
@@ -206,59 +209,62 @@ class ApiController extends ControllerBase{
         forward(uri: servletPath.replace('/*', "/$name"))
     }
 
-    /**
-     * API endpoints
-     */
-    /**
-     * Return true if grails configuration allows given feature, or '*' features
-     * @param name
-     * @return
-     */
-    private boolean featurePresent(def name){
-        boolean featureStatus = configurationService.getBoolean("feature.incubator.${name}", false)
 
-        def splat=configurationService.getBoolean("feature.incubator.*", false)
-        return splat || featureStatus
-    }
-    /**
-     * Set an incubator feature toggle on or off
-     * @param name
-     * @param enable
-     */
-    private void toggleFeature(def name, boolean enable){
-        grailsApplication.config.feature?.incubator?.putAt(name, enable)
-    }
+
+    @Operation(
+        method = "GET",
+        summary = "Get Rundeck System Feature Status",
+        description = "Return feature's on/off status",
+        parameters = [
+            @Parameter(
+                name='featureName',
+                in = ParameterIn.PATH,
+                description = 'Feature name without the `feature.` prefix, or blank to receive list of all system features',
+                allowEmptyValue = true,
+                required = false,
+                schema = @Schema(
+                    type='string'
+                )
+            )
+        ]
+    )
+    @ApiResponse(
+        responseCode = "200",
+        description = "Map of all system features' status if no specific one specified",
+        content = @Content(
+            mediaType = "application/json",
+            schema = @Schema(implementation = Map)
```

Comment:
There should be two `@ApiResponse`s, and each should point to a specific datatype for the `@Schema(implementation=DataType)`, instead of just Map.
One for the List result, and one for a FeatureEnabledResult type.

The Controller should also respond with the actual data type object.